explorer/Makefile

```diff
@@ -0,0 +1,26 @@
+# Copyright © 2022 Luther Systems, Ltd. All rights reserved.
+
+.DEFAULT_GOAL := up-clean
+
+.PHONY: up
+up: down
+	./check_for_peers.sh
+	docker-compose up -d
+
+.PHONY: watch
+watch: up
```

Comment:
I question the value of this target and suggest it just be removed.

If you'd rather keep this I would suggest it just be called `logs` instead of `watch` because I feel the latter implies that the logs would be streamed using `docker logs -f`, but that would lead to other issues...

I just don't think it's worth it as the underlying commands aren't hard to type and the number of times I would want the two logs concatenated seems minimal (especially if you assume they both work).

Reply:
That's fair, removed.